src/main/java/core/basesyntax/service/impl/ReportMakerServiceImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportMakerService;
+
+public class ReportMakerServiceImpl implements ReportMakerService {
+    @Override
+    public String createReport() {
+        StringBuilder report = new StringBuilder("fruit,quantity\n");
+        Storage.FRUIT_STORAGE
+                .forEach((key, value) -> report.append(key).append(",").append(value).append("\n"));
```

Comment:
I'd recommend you append `System.lineSeparator()` instead of `"\n"` cause it makes your code 100% multi-platformed